libs/utils/sidekick.js

```diff
@@ -22,17 +26,31 @@ export default function init({ createTag, loadBlock, loadScript, loadStyle }) {
     getModal(null, { id: 'preflight', content, closeEvent: 'closeModal' });
   };
 
+  const addVersion = async (event) => {
+    const url = `https://adobe.sharepoint.com/sites/adobecom/_api/web/GetFileByServerRelativeUrl('/sites/adobecom/CC/www${event.detail.data}.docx')`;
```

Comment:
This is hardcoded to `adobecom/CC`.  Is that intentional?

Reply:
@chrischrischris This is will be fixed.

Reply:
Fixed